plugins/amt/fu-plugin-amt.c

```diff
@@ -544,15 +553,35 @@ void
 fu_plugin_init (FuPlugin *plugin)
 {
 	fu_plugin_set_build_hash (plugin, FU_BUILD_HASH);
+	fu_plugin_alloc_data (plugin, sizeof (FuPluginData));
 }
 
 gboolean
 fu_plugin_coldplug (FuPlugin *plugin, GError **error)
 {
 	g_autoptr(FuDevice) dev = NULL;
-	dev = fu_plugin_amt_create_device (error);
+	dev = fu_plugin_amt_create_device (plugin, error);
 	if (dev == NULL)
 		return FALSE;
 	fu_plugin_device_add (plugin, dev);
 	return TRUE;
 }
+
+void
+fu_plugin_add_security_attrs (FuPlugin *plugin, FuSecurityAttrs *attrs)
+{
+	FuPluginData *data = fu_plugin_get_data (plugin);
+	g_autoptr(FwupdSecurityAttr) attr = NULL;
+
+	if (!fu_common_is_cpu_intel ())
+		return;
+
```

Comment:
Don't we need to keep track if there are any AMT devices detected too? remember this vfunc gets called even if fu_plugin_amt_create_device() never got called.

Reply:
yes, fixed